lizmap/modules/lizmap/classes/qgisExpressionUtils.class.php

```diff
@@ -89,28 +97,54 @@ public static function updateExpressionByUser($layer, $expression)
         }
```

Comment:
on the line above, it returns $expression, which does not exist anymore

Reply:
fixed